button-merchant/src/main/java/com/usebutton/merchant/ButtonApiImpl.java

```diff
@@ -74,9 +84,15 @@ static ButtonApi getInstance(String userAgent) {
         return buttonApi;
     }
 
+    private ButtonApiImpl(String userAgent) {
+        this.userAgent = userAgent;
+        this.sslManager = new SSLManager(CERTIFICATE_PATHS, null);
+    }
+
     @VisibleForTesting
-    ButtonApiImpl(String userAgent) {
+    ButtonApiImpl(String userAgent, SSLManager sslManager) {
```

Comment:
The `SSLManager` can get passed in via the `getInstance()` method. General speaking from the way the merchant library is currently setup I would probably recommend adding a `getInstance()` method on the `SSLManagerImpl` class that was suggested in the other comment and passing that instance in while the `ButtonApiImpl` class only knows about the interface `SSLManager`